packages/coinjoin/src/types/client.ts

```diff
@@ -1,9 +1,9 @@
-import { AllowedRange, Round, FeeRateMedians } from './coordinator';
+import { AllowedRange, CoordinatorFeeRate, FeeRateMedians, Round } from './coordinator';
 
 export interface CoinjoinStatusEvent {
     rounds: Round[];
     changed: Round[];
     feeRatesMedians: FeeRateMedians[];
-    coordinatorFeeRate: number;
+    coordinationFeeRate: CoordinatorFeeRate;
```

Comment:
You think this `coordinator/coordination` name swap is reasonable here? I was slightly confused at first.